content/influxdb/v2.0/reference/flux/stdlib/profiler/_index.md

```diff
@@ -74,3 +77,89 @@ include a table with a row for each operation and the following columns:
 - **MaxDuration:** maximum duration of the operation in nanoseconds
 - **DurationSum:** total duration of all operation executions in nanoseconds
 - **MeanDuration:** average duration of all operation executions in nanoseconds
+
+## Examples
+
+- [Measure query and operator performance for InfluxDB](#measure-query-and-operator-performance-for-influxdb)
+- [Measure query and operator performance of array.from()](#measure-query-and-operator-performance-of-arrayfrom)
```

Comment:
```suggestion
- [Measure query and operator performance of array.from()](#measure-query-and-operator-performance-of-array-from)
```

Reply:
When Hugo converts headings to anchors, it ignores periods and does not replace them with hyphens.